app/views/content/funding-and-support/salaried-teacher-training.md

```diff
@@ -0,0 +1,88 @@
+keywords:
+    - Teach First
+    - School Direct
+    - School Direct (salaried)
+    - Postgraduate teaching apprenticeship
+    - PGTA
+    - Salaried courses
+    - Salaried teacher training
+    - Salaried teacher training course
+    - Paid teacher training
+---
+
+There are some teacher training courses that come with a salary. 
+
+This means you will not have to pay tuition fees and will receive a salary while you train to get [qualified teacher status (QTS)](/what-is-qts). This is what you need to teach in many primary, secondary and special schools in England. 
+
+The main salaried courses include School Direct (salaried), postgraduate teaching apprenticeships (PGTA) and teacher training delivered by Teach First.
+
+All salaried courses are broadly the same, but the course length and qualifications you work towards may vary.
+
+They're in high demand and very competitive, so it’s important to apply as soon as you can if you’re eligible. 
+
+## Am I eligible for a salaried teacher training course? 
+
+For some salaried courses, teacher training providers may want you to have significant teaching or school experience (for example, if you’re already working as an unqualified teacher or teaching assistant).  
+
+Some also want you to have a school lined up to work in while you train. Speak to the teacher training provider to find out what their requirements are. 
+
+You can also [find out more about the qualifications you need to apply for all teacher training courses](/is-teaching-right-for-me/qualifications-you-need-to-teach). 
+
+Or talk to a [teacher training adviser](/teacher-training-advisers) who can help you understand which courses you could be eligible for. 
+
+## How much will I be paid on a salaried course? 
+
+You'll receive an [unqualified teacher’s salary](/salaries-and-benefits#unqualified-teacher-salaries) – the exact amount will vary depending on the school you work in. 
+
+It’s worth checking if you’re entitled to a [bursary or scholarship](/funding-and-support/scholarships-and-bursaries) before you apply for a salaried course. 
+
+If you do a salaried course, you will not be eligible for a bursary, scholarship or student finance. You’ll also be taxed on your income, whereas you will not be taxed on a bursary or scholarship. 
+
+If you’re eligible for a bursary or scholarship, you could receive more money on a non-salaried course than on a salaried one. 
+
+## How does salaried teacher training work? 
+
+Salaried teacher training is usually similar to non-salaried teacher training – you'll spend most of your time in school placements, with some theoretical learning. 
+
+You will not perform more than 90% of a full-time teacher’s duties (unless you’re on a postgraduate teaching apprenticeship, in which case it’s 80%). 
+
+Full-time salaried teacher training will usually last: 
+
+* 1 academic year (September to July) if you’re on a School Direct (salaried) course 
```

Comment:
@gemmadallmandfe @philspray know we've talked about removing academic year, but I feel like in this case because we validate it with months, it's okay?

Reply:
Tiny suggestion, probably makes no difference but would it be better the other way around? Putting the less familiar term in the brackets?

```suggestion
* from September to July (1 academic year) if you’re on a School Direct (salaried) course 
```

Reply:
I think your suggestion is good @philspray where there is only one year involved but not sure it would work for the 2 year Teach First bullet?

In which case, I think it is probably OK to leave as is, given that we've explained the months involved in brackets?